docs/sdk-beta/datastore.md

```diff
@@ -0,0 +1,47 @@
+# Datastore - COMING SOON
```

Comment:
Is there more information to be added?  If not, should we remove `COMING SOON`?

Reply:
yes will come in next PR going out today